src/improvements/tasks/sep/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,1042 @@
+# Validation
+
+This document can be used to validate the inputs and result of the execution of the upgrade transaction which you are
+signing.
+
+The steps are:
+
+1. [Validate the Domain and Message Hashes](#expected-domain-and-message-hashes)
+2. [Verifying the transaction input](#understanding-task-calldata)
+3. [Verifying the state changes](#state-changes)
+
+## Expected Domain and Message Hashes
+
+First, we need to validate the domain and message hashes. These values should match both the values on your ledger and
+the values printed to the terminal when you run the task.
+
+> [!CAUTION]
+>
+> Before signing, ensure the below hashes match what is on your ledger.
+>
+> ### Security Council
+>
+> - Domain Hash: ``
+> - Message Hash: ``
+>
+> ### Optimism Foundation
+>
+> - Domain Hash: ``
+> - Message Hash: ``
```

Comment:
```suggestion
> - Domain Hash: `0xbe081970e9fc104bd1ea27e375cd21ec7bb1eec56bfe43347c3e36c5d27b8533`
> - Message Hash: `0x94ce155b346db4cb94ba198fb251c0f6cddce09c36f46f764709db28fc409ef4`
```

Reply:
this will change because the nonces need to change.

Reply:
These hashes are for the security council not the foundation.

Reply:
I added the hashes in https://github.com/ethereum-optimism/superchain-ops/pull/709/commits/a9e2f5fd76501b3be700409f4b11fa17b629570a